src/users/api/v1/viewsets.py

```diff
@@ -3,7 +3,7 @@
 from rest_framework.views import APIView
 from rest_framework.response import Response
 from rest_framework import permissions, viewsets
-from .serializers import UserSerializer
+from ...serializers import UserSerializer
```

Comment:
sorry about not saying this earlier, but we want to reduce the chances of importing things in this manner.

```py
from ...serializers import UserSerializer
```

You would need to move the `serializers.py` file inside the `api/v1/` folder.